src/garage/tf/embeddings/gaussian_mlp_encoder.py

```diff
@@ -131,6 +131,22 @@ def _initialize(self):
             ],
             feed_list=[self.model.networks['default'].input])
 
+    def dist_info(self, input_val, state_infos):
```

Comment:
why not just provide an implementation?

Reply:
Because it is not used anywhere, and I want to defer its implementation until it is actually used in the future. This applies to other methods that I am not implementing

Reply:
i think if it's obvious how to provide the implementation, you should. these are reusable primitives, dist_info is implemented on every other stochastic primitive, and it's frustrating for a user to find a method with an obvious missing implementation.

remember that garage is more than the algorithm implementations. people should be able to use these primitives outside of garage.